lib/internal/Magento/Framework/App/Test/Unit/Cache/TypeListTest.php

```diff
@@ -6,21 +6,30 @@
 
 namespace Magento\Framework\App\Test\Unit\Cache;
 
-use \Magento\Framework\App\Cache\TypeList;
+use Magento\Framework\App\Cache\InstanceFactory;
+use Magento\Framework\App\Cache\StateInterface;
+use Magento\Framework\App\Cache\TypeList;
+use Magento\Framework\App\CacheInterface;
+use Magento\Framework\Cache\Frontend\Decorator\TagScope;
+use Magento\Framework\Cache\ConfigInterface;
+use Magento\Framework\DataObject;
 use Magento\Framework\Serialize\SerializerInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_MockObject_MockObject;
 
 /**
  * Test class for \Magento\Framework\App\Cache\TypeList
  */
-class TypeListTest extends \PHPUnit\Framework\TestCase
+class TypeListTest extends TestCase
 {
     /**
-     * @var \Magento\Framework\App\Cache\TypeList
+     * @var TypeList
      */
     protected $_typeList;
 
     /**
-     * @var \Magento\Framework\App\CacheInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var CacheInterface|PHPUnit_Framework_MockObject_MockObject
```

Comment:
Replace with `\PHPUnit\Framework\MockObject\MockObject`

Reply:
Hello @lbajsarowicz 
Thank you for your remark.
I have fixed it. Please check.
Thank you in advance.